src/lib/contentful/query/GetModuleInsightsBentoById.ts

```diff
@@ -0,0 +1,48 @@
+import { graphql } from "gql.tada";
+import { ComponentLinkFragment } from "../fragments/ComponentLinkFragment";
+import { RichTextFragments } from "../fragments/RichTextFragments.generated";
+import { ComponentInsightFragment } from "../fragments/ComponentInsightFragment";
+import { ComponentNewsletterSignupFragment } from "../fragments/ComponentNewsletterSignupFragment";
+
+export const GetModuleInsightsBentoById = graphql(
+  `
+    query GetModuleInsightsBentoById(
+      $locale: String
+      $preview: Boolean
+      $id: String!
+    ) {
+      moduleInsightsBento(id: $id, preview: $preview, locale: $locale) {
+        ...ModuleInsightsBentoFragment
+      }
+    }
+
+    fragment ModuleInsightsBentoFragment on ModuleInsightsBento {
+      sys {
+        id
+      }
+      headline {
+        ...ModuleInsightsBento_headlineFragment
+      }
+      eyebrow
+      subheading
+      exploreInsightsButton {
+        ...ComponentLinkFragment
+      }
+      insightsCollection {
+        items {
+          _id
```

Comment:
Do we know if this is the same as `sys { id }`?

Either way, if you're just using it to `key` React renders, I might argue that as long as we're never dynamically resorting the items, it's worth it to just key off the `index` and not bother to fetch for any extra data we don't need.